examples/nlp/glue_benchmark/glue_benchmark_config.yaml

```diff
@@ -40,14 +40,23 @@ model:
     num_samples: -1
     batch_size: 32
 
+
+  tokenizer:
+      tokenizer_name: ${model.language_model.pretrained_model_name} # or sentencepiece
```

Comment:
or sentencepiece or word or char - or refer to a method or a way to get the list of supported tokenizers

Reply:
there is get_tokenizer_list

Reply:
could we maybe add this to the comment instead of explicit listing?

Reply:
why? tokenizer_nameis a required input. This config is supposed to work out of the box, so it makes sense to use model.language_model.pretrained_model_name. which is BERT

Reply:
@titu1994  yes done in commit